app/src/combo.c

```diff
@@ -290,28 +290,34 @@ static int release_pressed_keys() {
 
 static inline int press_combo_behavior(const struct combo_cfg *combo, int32_t timestamp) {
     struct zmk_behavior_binding_event event = {
+        .binding = &combo->behavior,
+        .layer = 0, // Combos don't have layers, so their layer is set to be the base layer.
```

Comment:
That seems fair, though we could consider highest active layer that obey combo's layer filter.